package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@hotwired/stimulus": "^3.2.1",
+    "@hotwired/turbo-rails": "^7.3.0",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  }
+}
```

Comment:
- [ ] @Outkarm Great work setting up the project but you forgot to add `react-dom` and `babel-loader` dependencies in your package.json

Reply:
This change has been done well